DataFormats/L1TrackTrigger/interface/TTTrack.h

```diff
@@ -0,0 +1,521 @@
+/*! \class   TTTrack
+ *  \brief   Class to store the L1 Track Trigger tracks
+ *  \details After moving from SimDataFormats to DataFormats,
+ *           the template structure of the class was maintained
+ *           in order to accomodate any types other than PixelDigis
+ *           in case there is such a need in the future.
+ *
+ *  \author Nicola Pozzobon
+ *  \date   2013, Jul 12
+ *
+ */
+
+#ifndef L1_TRACK_TRIGGER_TRACK_FORMAT_H
+#define L1_TRACK_TRIGGER_TRACK_FORMAT_H
+
+#include "DataFormats/L1TrackTrigger/interface/TTStub.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/Phase2TrackerDigi/interface/Phase2TrackerDigi.h"
+
+template< typename T >
```

Comment:
Hi @sviret - since this is already templated - can it be simplified by templating on "4" and "5" or do both get used for the same object?